HRTool/src/main/java/app/controllers/ImageUploadRestController.java

```diff
@@ -14,55 +12,26 @@
 import org.springframework.web.bind.annotation.ResponseBody;
 import org.springframework.web.multipart.MultipartFile;
 
-
-
 @Controller
-
 public class ImageUploadRestController {
 
 	@Autowired
-    private ServletContext servletContext;
-
-	
-	@RequestMapping(value = "/upload", method = RequestMethod.GET)
-	public @ResponseBody String uploading(){
-		
-		
-		System.out.println("---------------------------------");
-		return "BLA BLA";
+	private ServletContext servletContext;
 
-
-	}
-	
 	@RequestMapping(value = "/upload", method = RequestMethod.POST)
-	public @ResponseBody String uploadingPOST(@RequestParam("flowFilename") String fileName,
-			@RequestParam("flowTotalChunks") int totalChunks,
-			@RequestParam("slika") MultipartFile file){
-		System.out.println("----------------POST-------------");
+	public @ResponseBody String uploadingPOST(
+			@RequestParam("picture") MultipartFile file) {
 		if (!file.isEmpty()) {
-			System.out.println("nije prazan  "+ fileName+ "  totalchunks: "+totalChunks);
-            try {
-            	
-                byte[] bytes = ((MultipartFile) file).getBytes();
-                BufferedOutputStream stream =
-                        new BufferedOutputStream(new FileOutputStream(
-                        		new File(servletContext.getRealPath("/images/")+fileName)));
-                stream.write(bytes);
-                stream.close();
-                System.out.println("---------------FILE UPLOADED------------" + servletContext.getRealPath("/images"));
-                return "images/"+fileName;
-            } catch (Exception e) {
-                return "You failed to upload " + e.getMessage();
-            }
-        } else {
-            return "You failed to upload  because the file was empty.";
-        }
+			try {
+				file.transferTo(new File(servletContext.getRealPath("/images")
+						+ file.getOriginalFilename()));
+				return "images/" + file.getOriginalFilename();
+			} catch (Exception e) {
+				return "You failed to upload " + e.getMessage();
```

Comment:
Never return error message in this format.
The appropriate format would be: return new ResponseEntity<>("<ERROR_ENUM>", HttpStatus.BadRequest) or something close.